example/lib/main.dart

```diff
@@ -143,8 +143,11 @@ class _MyAppState extends State<MyApp> {
         iOSAllowToolbarTitleChange: false,
         toolbarTitle: 'Custom Title',
         androidSettingsMenuItems:['theme', 'scrolldirection'],
+        iOSSettingsMenuItems:['scrollDirection', 'pageTransition', 'appearance', 'brightness', 'pageMode', 'spreadFitting'],
         showActionNavigationButtons: false,
-        iOSShowActionNavigationButtonLabels: false
+        iOSShowActionNavigationButtonLabels: false,
+        pageLayoutMode: 'automatic',
+        iOSIsFirstPageAlwaysSingle: false
```

Comment:
Because this configuration option does not have any effect when in single page mode, it would probably make sense to switch `pageLayoutMode` to double page.

Reply:
Makes sense!

Reply:
Addressed in https://github.com/PSPDFKit/pspdfkit-flutter/pull/46/commits/a6b4b208d0b9da548ff3301bdd496f3043fd674b